priv/download.sh

```diff
@@ -25,6 +25,42 @@ msg () {
     echo "$@" >&2
 }
 
+maybe_fix_dynamodblocal_dep_in_pom () {
+    POM_PATH="$1"
+    PKG_NAME="$2"
+    PKG_VSN="$3"
+
+    # "dynamodb-streams-kinesis-adapter" includes in its POM a test-ONLY dependency
+    # called "DynamoDBLocal". At the moment of writing (2023-10-27), the adapter
+    # release we are using (1.5.3), which was released in May 11, 2021, states that
+    # the test dependency should be downloaded in a version matching "[1.12,2.0)"
```

Comment:
oh my god someone actually USES math-style inclusive/exclusive bracketing? I suggested that to a friend who was working on a new language back in college and wanted to add range syntax, and he looked at me like I had 3 heads